doc-content/jbpm-docs/src/main/asciidoc/Kafka/kieserver-kafka-customformat-proc.adoc

```diff
@@ -20,5 +20,17 @@ You can download the interface definitons from the https://github.com/kiegroup/d
 +
 .. Provide the classes to your business application. For instructions, see xref:custom-class-provide-proc_{context}[].
 . Set the following {KIE_SERVER} system properties to set the custom writer or reader:
-** `org.kie.server.jbpm-kafka.ext.eventWriterClass`: the custom event writer class. Set this property to use a different format to send messages. If you want to use a custom format, set the property to the fully qualified name of your custom event writer class. If you want to use a raw JSON data format, set the property to `org.kie.server.services.jbpm.kafka.RawJsonEventWriter`.
-** `org.kie.server.jbpm-kafka.ext.eventReaderClass`: the custom event reader class. Set this property to use a different format to receive messages. If you want to use a custom format, set the property to the fully qualified name of your custom event reader class. If you want to use a raw JSON data format, set the property to `org.kie.server.services.jbpm.kafka.RawJsonEventReader`.
++
+.{KIE_SERVER} system properties for setting a custom writer or reader
+[cols="45%,55%", options="header"]
+|===
+|Property
+|Description
+
+|`org.kie.server.jbpm-kafka.ext.eventWriterClass`
+|The custom event writer class. Set this property to use a different format to send messages. If you want to use a custom format, set the property to the fully qualified name of your custom event writer class. If you want to use a raw JSON data format, set the property to `org.kie.server.services.jbpm.kafka.RawJsonEventWriter`.
```

Comment:
[Question] - Is "send", in this case, correct? Just considering the alternatives "post" and "emit".